src/encoded/static/components/schema.js

```diff
@@ -627,7 +627,12 @@ globals.contentViews.register(SchemaPage, 'JSONSchema');
 // add a new object of that type if you're logged in.
 const AllSchemasPage = (props, reactContext) => {
     const { context } = props;
```

Comment:
I saw this late. Since this is optional and I already have a rebased code-based ready for QA, I will address this is there are bugs on this ticket found or otherwise during a future enhancement.